lib/src/countly_config.dart

```diff
@@ -149,6 +149,12 @@ class CountlyConfig {
     return this;
   }
 
+  /// enable temporary ID mode
+  CountlyConfig enableTemporaryDeviceIDMode() {
+    _deviceID = Countly.deviceIDType['TemporaryDeviceID']!;
```

Comment:
"TemporaryDeviceID"
Did you changed value of this to 
"CLYTemporaryDeviceID" ?